lib/diagnostic.rb

```diff
@@ -115,10 +122,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person={age: 3, first_name: "Sheila"} '
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = 'person.default = "'Hi'+ #{first_name} + '.'"
```

Comment:
there are a number of ways to do this but with purely interpolation this is the way I would do it.
```rb
person.default = "#{person[:first_name]} <is great>
```